src/posts/2021-08-09.md

```diff
@@ -0,0 +1,59 @@
+
+## What did you do during your fellowship? What have you learned?
+Three projects come to mind. Once set up with the content management system (CMS) and the lavish suite of productivity apps, I spun up my developer environment in what I consider to be a shockingly painless process, and I was on my way to the first task. Dealt out at our sprint meeting on the first day after orientation, I was to rename an ambiguously worded text field in the backend interface for publishing liveblogs. Not to mention, our Django web framework needed to be updated to 2.2.24. Nothing on the cutting edge, but it did acquaint me with the stack (architecture and integrated modules of the website), as well as the file structure that I would be rummaging through many times to come.
+
+Next, in a meeting some days later, I learned about a pilot program in the data visuals department: the [Source Diversity Project](https://www.texastribune.org/2021/04/21/texas-tribune-diversity-report/). Possibly named à la the [BBC initiative](https://www.bbc.co.uk/5050) from which we’ve drawn some inspiration, this was the meat and potatoes of my work. The concept is that if we can collect enough data on the race, gender and ethnicity of the people we interview, we’ll be better equipped to produce coverage that is as representative of the Texan population as possible. However, keeping track of all those sources is not a simple undertaking. Understandably, not everyone submits these details to our internal Google Form at the first opportunity, and some may not be able to obtain the demographic information in the first place.
+
+The new workflow automation that I created is a Slackbot for keeping track of these stories whose source data could otherwise fall through the cracks. Previously, someone would have to spend considerable time manually comparing the website and a spreadsheet to see who had not filled out the Google Form. Convenience is the commodity that always sells, so naturally there was a motivation to abstract some of the menial tasks in this process.
+
+Now every Friday morning, our serverless Lambda script self-activates, cross-references the website and submission data and gently reminds reporters of recent outstanding articles with a personalized direct message. (The name of the game is to make life easier for the staff, not harder, so verbal or technical feedback is very much appreciated.) The hope is that an extra nudge here and there will keep everyone honest in our undertaking. Of course, as I conceded to the staff when I announced this automation, it may not be a riveting time to go back and enter all the data, but the time is well worth it for the greater effort. In a sense, I think, it matters not what action you are carrying out as a journalist in the day-to-day, but for whom you are carrying out said action.
+
```

Comment:
Well said! It might be a nice visual aid to include the Slack source reminder message screenshot from #test as an img tag here